.circleci/config.yml

```
@@ -391,6 +391,24 @@ jobs:
       - store_artifacts:
           path: /go/src/github.com/cosmos/gaia/gaia-windows-res.yml
 
+  contract-tests:
```

let's use underscore instead to keep consistency
```suggestion
  contract_tests:
```